demo_path_routing_auth/index.html

```diff
@@ -87,5 +87,40 @@ <h2>Upload your own data</h2>
       </div>
     </div>
 
+    <script>
+      (function() {
+        function set_parameters_to_default(tool_name) {
+          var default_parameters = {{ default_parameters_json | safe }};
+          var params_json = JSON.stringify(default_parameters[tool_name]);
+          document.getElementById('id_parameters_json').value = params_json;
+        }
+
+        function set_files_to_default(tool_name) {
```

Comment:
I wouldn't say short term: Hopefully the demo doesn't grow too large, or get to much use, but it will have a role as long as this project does.

I'll camel case it, and see how hard it would be to pull it into a separate file.

Reply:
@jkmarx : Remembered the other reason I have this inline: JS datastructures are populated by template interpolations:
```
function setParametersToDefault(toolName) {
          var defaultParameters = {{ default_parameters_json | safe }};
         ...
```

It could be done other ways:
- Pull the rest of the JS out, but set these as globals in the index.html template.
- Have the JS hit a mini-API that serves JSON.

... but for something as small as this, I think that either of those would actually make it less maintainable, but I could be wrong.